crates/router/src/core/payments/transformers.rs

```diff
@@ -67,17 +68,11 @@ where
             connector_metadata: None,
         });
 
-    let router_return_url = Some(helpers::create_redirect_url(
-        &state.conf.server,
-        &payment_data.payment_attempt,
-        connector_id,
-        payment_data.creds_identifier.as_deref(),
-    ));
-    let complete_authorize_url = Some(helpers::create_complete_authorize_url(
-        &state.conf.server,
-        &payment_data.payment_attempt,
-        &connector_id.to_owned(),
-    ));
+    let additional_data = PaymentAdditionalData {
+        base_url: state.conf.server.base_url.clone(),
```

Comment:
```suggestion
        router_base_url: state.conf.server.base_url.clone(),
```

Reply:
Other places also prefer `router_base_url`

Reply:
changed it to router_base_url at all other places